aptos-move/framework/aptos-framework/sources/staking_contract.spec.move

```diff
@@ -385,24 +386,23 @@ spec aptos_framework::staking_contract {
         aborts_if !exists<Store>(staker_address) && !exists<account::Account>(staker_address);
         aborts_if !exists<Store>(staker_address) && account.guid_creation_num + 9 >= account::MAX_GUID_CREATION_NUM;
         ensures exists<Store>(staker_address);
-
-        let store = global<Store>(staker_address);
-        let staking_contracts = store.staking_contracts;
-        aborts_if simple_map::spec_contains_key(staking_contracts, operator);
-
-        // verify create_stake_pool()
-        let seed_0 = bcs::to_bytes(staker_address);
-        let seed_1 = concat(concat(concat(seed_0, bcs::to_bytes(operator)), SALT), contract_creation_seed);
-        let resource_addr = account::spec_create_resource_address(staker_address, seed_1);
-        include CreateStakePoolAbortsIf {resource_addr};
-
-        // verify stake::add_stake_with_cap()
-        include StakeAddStakeWithCapAbortsIf{ pool_address: resource_addr };
-
+        //
```

Comment:
Is this comment block intended?

Reply:
Yes. I need to verify other two high-level properties defined in it so they are commented out because of timeout.

Reply:
I see. Could we put a TODO together with this reason for future reference?